src/plugin/vk-wrapper/graphicsPipeline/GraphicsPipeline.hpp

```diff
@@ -0,0 +1,77 @@
+/**************************************************************************
+ * VkWrapper v0.0.0
+ *
+ * VkWrapper is a software package, part of the Engine².
+ *
+ * This file is part of the VkWrapper project that is under GPL-3.0 License.
+ * Copyright © 2024 by @MasterLaplace, All rights reserved.
```

Comment:
Yes, you did the PR, but you are not the only contributor in this project. Use the organizations handler @EngineSquared